0019-glossary/0019-glossary.md

```diff
@@ -0,0 +1,248 @@
+To be fully Interledger-compatible, ledgers MUST support conditional transfers, though it is possible to send Interledger payments over a ledger that does not natively support the recommended features. See [IL-RFC 17](../0017-ledger-requirements/0017-ledger-requirements.md) for the full description and tiers of ledger requirements.
+
+## Connector
+A party who chains one transfer to the next in an interledger payment.
+A connector receives a local transfer on one ledger in exchange for making another local transfer on a different ledger. A single interledger payment may include multiple connectors and may traverse any number of ledgers.
+Connectors take some risk, but this risk can be managed and is primarily based upon the connector's chosen ledgers and direct peers.
+
+## Cyclic Transaction
+A transaction where the destination account is the same account, on the same ledger, as the source account. This can be useful
+when rebalancing liquidity (to enable future payments), or when rebalancing stored value (to spread risk, or to take advantage
+of changing exchange rates).
+
+# Destination Account
+The account of the receiver, whose address is included in the
+[interledger packet](#interledger-packet).
+Note that anyone can claim to have a certain [Interledger address](#interledger-address); address ownership is not enforced.
+
+# Destination Amount
+The amount to be received by the receiver.
+
+# Destination Ledger
+The ledger on which the final receiver of an Interledger payment holds their account
+
+## Distributed Ledger
+A ledger that is operated by a group of entities and/or run on multiple servers. This term is used somewhat interchangeably with "Blockchain" or "Decentralized Ledger".
+
+## Exchange Rate
+The price of one ledger's asset in terms of another ledger's asset.
+Connectors may generate revenue from the difference in value between incoming and outgoing transfers.
+Senders may request quotes from multiple connectors to determine the best price before sending a payment.
+The exchange rate between source and destination is determined by the product of exchange rates at each hop.
+
+## Executed
+A final state of a transfer indicating that the funds were transferred to the account of the immediate recipient.
+
+## Execution Phase
+The second phase of an ILP payment, in which the fulfillment is passed back from the receiver, via the connector(s), to the sender.
+
+## Final State
+The state an interledger payment ends up in. Has to be either
+[rejected](#rejected) or [executed](#executed).
+
+## Fulfillment
+See [conditional transfer](#conditional-transfer).
```

Comment:
Should be "A 32-byte value used to trigger the execution of a transfer. In most Interledger payments, the fulfillment is known only to the receiver (or in the case of the Pre-Shared Key protocol it is known to the sender and the receiver). For more details see conditional transfer"